requirements.txt

```diff
@@ -1,3 +1,4 @@
 django
```

Comment:
Add a version dependency here for 4.1 - DjangoRestFramework doesn't yet support 4.2